crates/wasmi/src/store.rs

```diff
@@ -67,6 +70,28 @@ impl StoreIdx {
 /// A stored entity.
 pub type Stored<Idx> = GuardedEntity<StoreIdx, Idx>;
 
+pub struct ResourceLimiterRef<'a>(Option<&'a mut (dyn crate::ResourceLimiter)>);
```

Comment:
I'd love to see a (short) doc comment explaining the rational behind the existence of this type for our future selves. :)

Reply:
Done.